packages/experimental/Divider/SPEC.md

```diff
@@ -153,6 +153,31 @@ export interface DividerTokens extends LayoutTokens {
    * @default 8
    */
   minLineSize?: number;
+  /**
+   * If text is rendered in the divider, this flag controls whether it is italicized or not.
+   */
+  textItalic?: TextProps['italic'];
```

Comment:
There's a shared FontTokens type already which has fontWeight, fontSize, and variant, so it might be better to move italic/underline/strikethrough there and then extend DividerTokens to use FontTokens.